support-frontend/assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -160,6 +160,7 @@ const getProducts = (
 
 const getProduct = (subsLinks: SubsUrls, countryGroupId: CountryGroupId): ?Product => {
   const products = getProducts(subsLinks, countryGroupId);
+  const promoteGuardianWeekly = true;
```

Comment:
I think it might be simpler to remove this variable and the if clause that relates to it, then also remove 
```
if (countryGroupId === GBPCountries) {
        return products.Paper;	
  }
```	      
that would give us the behaviour we want with a lot fewer ifs to read through?

Reply:
Based on the card, I thought what we wanted was for two weeks to push GW and then have another PR to remove that and return to existing behaviour. Based on standup, I think we might just want 
```
default: 
    return products.GuardianWeekly;
```

But I shall find that out

Reply:
I think that's right except that we want to keep digipack as long as the flash sale is on, so if you left that if clause in it would mean you could merge now and it would do the right thing

Reply:
If we ordinarily need to maintain digital pack behaviour during a flash sale, then that's why we need the `const promoteGuardianWeekly` because we are currently running a digital pack flash sale but want to promote GW instead this time.

Reply:
OK I misunderstood the requirements and you're totally right @rupertbates 

Thanks for your patience 😄